src/Microsoft.AspNetCore.Mvc.Razor.Extensions/ModelDirective.cs

```diff
@@ -94,17 +94,6 @@ protected override void ExecuteCore(RazorCodeDocument codeDocument, DocumentInte
                 }
 
                 var baseType = visitor.Class?.BaseType?.Replace("<TModel>", "<" + modelType + ">");
-                for (var i = visitor.InheritsDirectives.Count - 1; i >= 0; i--)
```

Comment:
This is unnecessary code. This was not getting executed because they will be no Inherits directives left at this point before this change. The unit tests were incorrectly verifying this code instead of the line above.